### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by denys-hrulov**
Added a lot of tests and deleted some unused features while testing

### CODE REVIEW

File: tests/TestDatePriortyView.cpp [hunk 1]
```diff
@@ -67,3 +67,44 @@ TEST_F(PriorityViewTest, AllTasksSortedByDate) {
...
+    DatePriorityView tw;
+    tw.addToView(node1);
+    tw.addToView(node2);
+    tw.addToView(node3);
+    tw.addToView(node4);
+    std::vector nodes = {node1, node2, node3, node4};
+    std::sort(nodes.begin(), nodes.end(),
+            [] (const auto& lhs, const auto& rhs) {
+                return lhs->getTask().getDate() < rhs->getTask().getDate();
+    });
```
Comment (IlliaDmitriiev) [inline tests/TestDatePriortyView.cpp:81]: Why do you use sort ? I hope it's only for cheking data

Comment (denys-hrulov) [reply tests/TestDatePriortyView.cpp:81]: We need sort to make our tasks ordered by date for sure. If we do not the test of View does not make sense.

Comment (denys-hrulov) [reply tests/TestDatePriortyView.cpp:81]: *edit. We FIRST add them to View and THEN sort.

Comment (IlliaDmitriiev) [reply tests/TestDatePriortyView.cpp:81]: got it

**Review by IlliaDmitriiev — APPROVED** [review]

**Pull request merged**